lib/index.js

```diff
@@ -11,10 +11,9 @@ require('useragent/features');                      // Enhances Useragent
 const internals = {};
 
 
-exports.register = (server, options, next) => {
+exports.register = async function (server, options) {
 
-    server.ext('onRequest', internals.onRequest);
-    return next();
+    await server.ext('onRequest', internals.onRequest);
```

Comment:
`server.ext` is not async, so you don't need to await, and you don't need register to be async.